lib/diagnostic.rb

```diff
@@ -108,10 +108,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = { :favorite_number => 10, :first_name => "Joe"}'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'my name is #{person.values[1]}I count to #{person.values[0]}'
```

Comment:
This is nearly right.

`'person.default = "my name is #{person[:first_name]} I count to #{person[:favorite_number]}"'`